litezip/completezip.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from pathlib import Path
+
+from .main import parse_litezip
+
+
+__all__ = (
+    'convert_completezip',
+)
+
+
+def convert_completezip(path):
+    """Converts a completezip file structure to a litezip file structure.
+    Returns a litezip data structure.
+
+    """
+    for filepath in path.glob('**/index_auto_generated.cnxml'):
```

Comment:
😠  Yes, I have this correctly written in the planning document. Something in our more recent conversations changed my thoughts on this. But it is correct to use the computed one rather than the user supplied version.

> The expansion part of this will delete the index.cnxml file and replace it with the index_auto_generated.cnxml file if it exists.